footer.js

```diff
@@ -0,0 +1,60 @@
+import * as idb from 'idb-keyval';
+
+async function fetchGithubContributors(requestURL) {
+  let response;
+  if ('caches' in window) {
+    response = await caches.match(requestURL);
+    if (!response) {
+      response = await fetch(requestURL);
+      const cache = await caches.open(requestURL);
+      await cache.put(requestURL, response.clone());
+    }
+  } else {
+    // browser does not support the cache APi
+    response = await fetch(requestURL);
+  }
+
+  response = await fetch(requestURL);
```

Comment:
@TakeAWalk I'm making `fetch` here a more complicated than it needs to be, it's normally just
```javascript
const response = await fetch(url);
const fetchedObject = await response.json();
// do stuff with fetchedObject
```
You can see how there is a lot less setup when compared to `XMLHttpRequest()`.

I'm caching the result in too ways, it's not much of a performance benefit with such a small amount of data, it's just a habit I'm trying to get into.